m2m-transfers/m2m-transfers-json-contract.md

```diff
@@ -180,7 +183,7 @@ list of institution and member limits, eligible shares, list of scheduled transf
   "rgState": "CREATETRAN",
   "powerOnFileName": "BANNO.M2MTRANSFERS.V1.POW",
   "userChrList": [
-    { "id": 1, "value": "1234567890S0001|9876543210|L0001|weekly|12/31/2021|1|15" },  // [sourceAccount][recipient member id][account type or id][frequency]|[startDate]|[day1]|[day2]
+    { "id": 1, "value": "1234567890S0001|9876543210|L0001|weekly|12/31/2021|1|15" },  // [sourceAccount][recipient member id][account type or id][frequency]|[startDate || "soonest" ]|[day1]|[day2]
```

Comment:
On freq.. looks like that is what is happening. Maybe change the "||" to "or" since "|" is the delimiter and it could be confusing during future updates?

Reply:
will be changed to "or"